data/decks/noel.json

```diff
@@ -0,0 +1,179 @@
+{
+	"id": "noel",
+	"season": 5,
+	"display_name": "Noel Vermillion",
+	"cardback": "cardback_noel.jpg",
+	"exceed_cost": 4,
+	"exceed_to_overdrive": true,
+	"on_exceed": {
+		"effect_type": "pass"
+	},
+	"overdrive_effect": {
+		"effect_type": "choice",
+		"choice": [
+			{ "effect_type": "close", "amount": 1 },
+			{ "effect_type": "retreat", "amount": 1 }
+		]
+	},
+	"ability_effects": [
+		{
+			"timing": "on_reshuffle",
+			"condition": "manual_reshuffle",
+			"effect_type": "add_set_aside_card_to_deck",
+			"id": "noel_valkyrieveil"
```

Comment:
make sure to add this as an exceed ability effect as well (we forgot in the existing s5 files, should be fixed soon)